src/main/takeDetection/takeDetection.ts

```diff
@@ -0,0 +1,104 @@
+import { Word, IndexRange } from 'sharedTypes';
+import {
+  InjectableTake,
+  InjectableTakeGroup,
+} from 'main/editDelete/injectTakeInfo';
```

Comment:
The import path seems wrong, it should be `../../sharedTypes` and `../../main/editDelete/injectTakeInfo`